### ISSUE TITLE
error: unexpected token

### ISSUE THREAD

**Issue by junosuarez** [zacanger/dist-upgrade#4]
I just tried running this to get the new npm that was just release, and got:
```
> npx dist-upgrade
npx: installed 508 in 12.981s
Unexpected token .
```

**Comment by junosuarez** [comment]
I `npm i -g dist-upgrade` to see if that made a difference, and I get the same error, but a much more useful output:
```
> dist-upgrade
/usr/local/lib/node_modules/dist-upgrade/index.js:5
var arg.trim() = process.argv[2]
       ^

SyntaxError: Unexpected token .
    at createScript (vm.js:74:10)
    at Object.runInThisContext (vm.js:116:10)
    at Module._compile (module.js:533:28)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Function.Module.runMain (module.js:605:10)
    at startup (bootstrap_node.js:158:16)
    at bootstrap_node.js:575:3
```

**Comment by zautumnz** [comment]
Whoops! Dumb typo on my part.

**Comment by zautumnz** [comment]
Fixed in 0.2.1
